`make run-lab` not working

**Describe the bug**



**How to fix?**
- [ ] replace ``echo "Starting of `run-lab`"`` to the `echo "Starting of 'run-lab'` [tag-v1.0.0 - Makefile#L27](https://github.com/MistyDi/data_analytics-math_python_pandas/blob/24d098116d6dd91c86f8e322483e1b67fdf204c0/Makefile#L27)

**How to overcome while not fixed?**
- start jupyter-lab manually

**To Reproduce**
Steps to reproduce the behavior:
1. move to the cloned repo-dir
2. run `C:\Apps\cygwin64\bin\make.exe run-lab` in the shell

**Expected behavior**

Building and dependencies installing is successful, jupyter-lab launched with the jupyter-notebook installed.

**Additional context**

<details>
  <summary>OS Details</summary><pre>
Название ОС:                      Майкрософт Windows 11 Домашняя для одного языка
Версия ОС:                        10.0.22000 Н/Д построение 22000
Изготовитель ОС:                  Microsoft Corporation
Параметры ОС:                     Изолированная рабочая станция
Сборка ОС:                        Multiprocessor Free
Зарегистрированная организация:   HP
Дата установки:                   23.10.2021, 21:25:06
Время загрузки системы:           23.05.2022, 13:19:02
Изготовитель системы:             HP
Модель системы:                   HP ENVY x360 Convertible 15-ee0xxx
Тип системы:                      x64-based PC
Процессор(ы):                     Число процессоров - 1.
                                  [01]: AMD64 Family 23 Model 96 Stepping 1 AuthenticAMD ~2000 МГц
Версия BIOS:                      Insyde F.19, 23.08.2021
Папка Windows:                    C:\WINDOWS
Системная папка:                  C:\WINDOWS\system32
Язык системы:                     ru;Русский
Язык ввода:                       en-us;Английский (США)
Полный объем физической памяти:   15 757 МБ
Доступная физическая память:      3 950 МБ
Виртуальная память: Макс. размер: 18 829 МБ
Виртуальная память: Доступна:     2 336 МБ
Виртуальная память: Используется: 16 493 МБ
</pre></details>

```text
 C:\Apps\cygwin64\bin\make.exe run-lab
echo "Starting of 'pip-update'"
Starting of 'pip-update'
pip install --upgrade pip --user
Requirement already satisfied: pip in c:\program files\python310\lib\site-packages (22.1.1)
echo "Starting of 'build-lab'"
Starting of 'build-lab'
pip install jupyterlab --user  # installing globally for the user
Requirement already satisfied: jupyterlab in c:\users\dmitm\appdata\roaming\python\python310\site-packages (3.4.2)
Requirement already satisfied: jupyter-server~=1.16 in c:\program files\python310\lib\site-packages (from jupyterlab) (1.17.0)
Requirement already satisfied: nbclassic~=0.2 in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from jupyterlab) (0.3.7)
Requirement already satisfied: ipython in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from jupyterlab) (8.3.0)
Requirement already satisfied: jupyterlab-server~=2.10 in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from jupyterlab) (2.14.0)
Requirement already satisfied: packaging in c:\program files\python310\lib\site-packages (from jupyterlab) (21.3)
Requirement already satisfied: jupyter-core in c:\program files\python310\lib\site-packages (from jupyterlab) (4.10.0)
Requirement already satisfied: jinja2>=2.1 in c:\program files\python310\lib\site-packages (from jupyterlab) (3.1.2)
Requirement already satisfied: tornado>=6.1.0 in c:\program files\python310\lib\site-packages (from jupyterlab) (6.1)
Requirement already satisfied: MarkupSafe>=2.0 in c:\program files\python310\lib\site-packages (from jinja2>=2.1->jupyterlab) (2.1.1)
Requirement already satisfied: nbconvert>=6.4.4 in c:\program files\python310\lib\site-packages (from jupyter-server~=1.16->jupyterlab) (6.5.0)
Requirement already satisfied: pywinpty in c:\program files\python310\lib\site-packages (from jupyter-server~=1.16->jupyterlab) (2.0.5)
Requirement already satisfied: prometheus-client in c:\program files\python310\lib\site-packages (from jupyter-server~=1.16->jupyterlab) (0.14.1)
Requirement already satisfied: anyio<4,>=3.1.0 in c:\program files\python310\lib\site-packages (from jupyter-server~=1.16->jupyterlab) (3.6.1)
Requirement already satisfied: traitlets>=5.1 in c:\program files\python310\lib\site-packages (from jupyter-server~=1.16->jupyterlab) (5.2.1.post0)
Requirement already satisfied: jupyter-client>=6.1.12 in c:\program files\python310\lib\site-packages (from jupyter-server~=1.16->jupyterlab) (7.3.1)
Requirement already satisfied: pyzmq>=17 in c:\program files\python310\lib\site-packages (from jupyter-server~=1.16->jupyterlab) (23.0.0)
Requirement already satisfied: terminado>=0.8.3 in c:\program files\python310\lib\site-packages (from jupyter-server~=1.16->jupyterlab) (0.15.0)
Requirement already satisfied: websocket-client in c:\program files\python310\lib\site-packages (from jupyter-server~=1.16->jupyterlab) (1.3.2)
Requirement already satisfied: argon2-cffi in c:\program files\python310\lib\site-packages (from jupyter-server~=1.16->jupyterlab) (21.3.0)
Requirement already satisfied: nbformat>=5.2.0 in c:\program files\python310\lib\site-packages (from jupyter-server~=1.16->jupyterlab) (5.4.0)
Requirement already satisfied: Send2Trash in c:\program files\python310\lib\site-packages (from jupyter-server~=1.16->jupyterlab) (1.8.0)
Requirement already satisfied: pywin32>=1.0 in c:\program files\python310\lib\site-packages (from jupyter-core->jupyterlab) (304)
Requirement already satisfied: json5 in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from jupyterlab-server~=2.10->jupyterlab) (0.9.8)
Requirement already satisfied: requests in c:\program files\python310\lib\site-packages (from jupyterlab-server~=2.10->jupyterlab) (2.27.1)
Requirement already satisfied: babel in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from jupyterlab-server~=2.10->jupyterlab) (2.10.1)
Requirement already satisfied: jsonschema>=3.0.1 in c:\program files\python310\lib\site-packages (from jupyterlab-server~=2.10->jupyterlab) (4.5.1)
Requirement already satisfied: notebook-shim>=0.1.0 in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from nbclassic~=0.2->jupyterlab) (0.1.0)
Requirement already satisfied: notebook<7 in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from nbclassic~=0.2->jupyterlab) (6.4.11)
Requirement already satisfied: setuptools>=18.5 in c:\program files\python310\lib\site-packages (from ipython->jupyterlab) (58.1.0)
Requirement already satisfied: stack-data in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from ipython->jupyterlab) (0.2.0)
Requirement already satisfied: jedi>=0.16 in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from ipython->jupyterlab) (0.18.1)
Requirement already satisfied: decorator in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from ipython->jupyterlab) (5.1.1)
Requirement already satisfied: matplotlib-inline in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from ipython->jupyterlab) (0.1.3)
Requirement already satisfied: pygments>=2.4.0 in c:\program files\python310\lib\site-packages (from ipython->jupyterlab) (2.12.0)
Requirement already satisfied: pickleshare in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from ipython->jupyterlab) (0.7.5)
Requirement already satisfied: backcall in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from ipython->jupyterlab) (0.2.0)
Requirement already satisfied: prompt-toolkit!=3.0.0,!=3.0.1,<3.1.0,>=2.0.0 in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from ipython->jupyterlab) (3.0.29)
Requirement already satisfied: colorama in c:\program files\python310\lib\site-packages (from ipython->jupyterlab) (0.4.4)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in c:\program files\python310\lib\site-packages (from packaging->jupyterlab) (3.0.9)
Requirement already satisfied: idna>=2.8 in c:\program files\python310\lib\site-packages (from anyio<4,>=3.1.0->jupyter-server~=1.16->jupyterlab) (3.3)
Requirement already satisfied: sniffio>=1.1 in c:\program files\python310\lib\site-packages (from anyio<4,>=3.1.0->jupyter-server~=1.16->jupyterlab) (1.2.0)

Requirement already satisfied: parso<0.9.0,>=0.8.0 in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from jedi>=0.16->ipython->jupyterlab) (0.8.3)
Requirement already satisfied: attrs>=17.4.0 in c:\program files\python310\lib\site-packages (from jsonschema>=3.0.1->jupyterlab-server~=2.10->jupyterlab) (21.4.0)
Requirement already satisfied: pyrsistent!=0.17.0,!=0.17.1,!=0.17.2,>=0.14.0 in c:\program files\python310\lib\site-packages (from jsonschema>=3.0.1->jupyterlab-server~=2.10->jupyterlab) (0.18.1)
Requirement already satisfied: python-dateutil>=2.8.2 in c:\program files\python310\lib\site-packages (from jupyter-client>=6.1.12->jupyter-server~=1.16->jupyterlab) (2.8.2)
Requirement already satisfied: entrypoints in c:\program files\python310\lib\site-packages (from jupyter-client>=6.1.12->jupyter-server~=1.16->jupyterlab) (0.4)
Requirement already satisfied: nest-asyncio>=1.5.4 in c:\program files\python310\lib\site-packages (from jupyter-client>=6.1.12->jupyter-server~=1.16->jupyterlab) (1.5.5)
Requirement already satisfied: beautifulsoup4 in c:\program files\python310\lib\site-packages (from nbconvert>=6.4.4->jupyter-server~=1.16->jupyterlab) (4.11.1)
Requirement already satisfied: jupyterlab-pygments in c:\program files\python310\lib\site-packages (from nbconvert>=6.4.4->jupyter-server~=1.16->jupyterlab) (0.2.2)
Requirement already satisfied: pandocfilters>=1.4.1 in c:\program files\python310\lib\site-packages (from nbconvert>=6.4.4->jupyter-server~=1.16->jupyterlab) (1.5.0)
Requirement already satisfied: tinycss2 in c:\program files\python310\lib\site-packages (from nbconvert>=6.4.4->jupyter-server~=1.16->jupyterlab) (1.1.1)
Requirement already satisfied: defusedxml in c:\program files\python310\lib\site-packages (from nbconvert>=6.4.4->jupyter-server~=1.16->jupyterlab) (0.7.1)
Requirement already satisfied: mistune<2,>=0.8.1 in c:\program files\python310\lib\site-packages (from nbconvert>=6.4.4->jupyter-server~=1.16->jupyterlab) (0.8.4)
Requirement already satisfied: nbclient>=0.5.0 in c:\program files\python310\lib\site-packages (from nbconvert>=6.4.4->jupyter-server~=1.16->jupyterlab) (0.6.3)
Requirement already satisfied: bleach in c:\program files\python310\lib\site-packages (from nbconvert>=6.4.4->jupyter-server~=1.16->jupyterlab) (5.0.0)
Requirement already satisfied: fastjsonschema in c:\program files\python310\lib\site-packages (from nbformat>=5.2.0->jupyter-server~=1.16->jupyterlab) (2.15.3)
Requirement already satisfied: ipython-genutils in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from notebook<7->nbclassic~=0.2->jupyterlab) (0.2.0)
Requirement already satisfied: ipykernel in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from notebook<7->nbclassic~=0.2->jupyterlab) (6.13.0)
Requirement already satisfied: wcwidth in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from prompt-toolkit!=3.0.0,!=3.0.1,<3.1.0,>=2.0.0->ipython->jupyterlab) (0.2.5)
Requirement already satisfied: argon2-cffi-bindings in c:\program files\python310\lib\site-packages (from argon2-cffi->jupyter-server~=1.16->jupyterlab) (21.2.0)
Requirement already satisfied: pytz>=2015.7 in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from babel->jupyterlab-server~=2.10->jupyterlab) (2022.1)
Requirement already satisfied: certifi>=2017.4.17 in c:\program files\python310\lib\site-packages (from requests->jupyterlab-server~=2.10->jupyterlab) (2022.5.18.1)
Requirement already satisfied: charset-normalizer~=2.0.0 in c:\program files\python310\lib\site-packages (from requests->jupyterlab-server~=2.10->jupyterlab) (2.0.12)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in c:\program files\python310\lib\site-packages (from requests->jupyterlab-server~=2.10->jupyterlab) (1.26.9)
Requirement already satisfied: executing in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from stack-data->ipython->jupyterlab) (0.8.3)
Requirement already satisfied: asttokens in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from stack-data->ipython->jupyterlab) (2.0.5)
Requirement already satisfied: pure-eval in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from stack-data->ipython->jupyterlab) (0.2.2)
Requirement already satisfied: six>=1.5 in c:\program files\python310\lib\site-packages (from python-dateutil>=2.8.2->jupyter-client>=6.1.12->jupyter-server~=1.16->jupyterlab) (1.16.0)
Requirement already satisfied: cffi>=1.0.1 in c:\program files\python310\lib\site-packages (from argon2-cffi-bindings->argon2-cffi->jupyter-server~=1.16->jupyterlab) (1.15.0)
Requirement already satisfied: soupsieve>1.2 in c:\program files\python310\lib\site-packages (from beautifulsoup4->nbconvert>=6.4.4->jupyter-server~=1.16->jupyterlab) (2.3.2.post1)
Requirement already satisfied: webencodings in c:\program files\python310\lib\site-packages (from bleach->nbconvert>=6.4.4->jupyter-server~=1.16->jupyterlab) (0.5.1)
Requirement already satisfied: psutil in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from ipykernel->notebook<7->nbclassic~=0.2->jupyterlab) (5.9.1)
Requirement already satisfied: debugpy>=1.0 in c:\users\dmitm\appdata\roaming\python\python310\site-packages (from ipykernel->notebook<7->nbclassic~=0.2->jupyterlab) (1.6.0)
Requirement already satisfied: pycparser in c:\program files\python310\lib\site-packages (from cffi>=1.0.1->argon2-cffi-bindings->argon2-cffi->jupyter-server~=1.16->jupyterlab) (2.21)
echo "Starting of 'build'"
Starting of 'build'
pipenv install
Creating a virtualenv for this project...
Pipfile: C:\Data\Job Tests\MSP_Bank-Math-Python-Pandas\Pipfile
Using C:/Program Files/Python310/python.exe (3.10.4) to create virtualenv...
[    ] Creating virtual environment...created virtual environment CPython3.10.4.final.0-64 in 486ms
  creator CPython3Windows(dest=C:\Users\dmitm\.virtualenvs\MSP_Bank-Math-Python-Pandas-fdDMlA_M, clear=False, no_vcs_ignore=False, global=False)
  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=C:\Users\dmitm\AppData\Local\pypa\virtualenv)
    added seed packages: pip==22.0.4, setuptools==62.1.0, wheel==0.37.1
  activators BashActivator,BatchActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator

Successfully created virtual environment!
Virtualenv location: C:\Users\dmitm\.virtualenvs\MSP_Bank-Math-Python-Pandas-fdDMlA_M
Installing dependencies from Pipfile.lock (677338)...
  ================================ 36/36 - 00:01:42
To activate this project's virtualenv, run pipenv shell.
Alternatively, run a command inside the virtualenv with pipenv run.
echo "Starting of 'build_kernel'"
Starting of 'build_kernel'
pipenv run python -m ipykernel install --user --name="pipenv-data_analytics_test"
Installed kernelspec pipenv-data_analytics_test in C:\Users\dmitm\AppData\Roaming\jupyter\kernels\pipenv-data_analytics_test
echo "Starting of `run-lab`"
/bin/sh: run-lab: command not found
Starting of
jupyter lab "data_analytics_task.ipynb"
usage: jupyter [-h] [--version] [--config-dir] [--data-dir] [--runtime-dir] [--paths] [--json] [--debug] [subcommand]

Jupyter: Interactive Computing

positional arguments:
  subcommand     the subcommand to launch

options:
  -h, --help     show this help message and exit
  --version      show the versions of core jupyter packages and exit
  --config-dir   show Jupyter config dir
  --data-dir     show Jupyter data dir
  --runtime-dir  show Jupyter runtime dir
  --paths        show all Jupyter paths. Add --json for machine-readable format.
  --json         output paths as machine-readable json
  --debug        output debug information about paths

Available subcommands: dejavu execute kernel kernelspec migrate nbconvert run server troubleshoot trust

Jupyter command `jupyter-lab` not found.
make: *** [Makefile:27: run-lab] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -24,7 +24,7 @@ run : build-notebook build_kernel
 	jupyter notebook $(D_NOTEBOOK)
 
 run-lab : build-lab build_kernel
-	echo "Starting of `run-lab`"
+	echo "Starting of 'run-lab'"
 	jupyter lab $(D_NOTEBOOK)
 
 update:

